students.html

```diff
@@ -0,0 +1,148 @@
+---
```

Comment:
Is removing the students and putting it into a seperate file required? The overall goal of the site is to display the students who took a part in GCI, putting it in a different page won't help. To reduce the site loading time, we could improve JavaScript.